### PR TITLE
tools/rados: add an option "--input-file file" to `setomapval` command

### PR DESCRIPTION
**Pull request by Songweibin**
Signed-off-by: songweibin <song.weibin@zte.com.cn>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/tools/rados/rados.cc [hunk 1]
```diff
@@ -189,6 +189,7 @@ void usage(ostream& out)
 "   -s name\n"
 "   --snap name\n"
 "        select given snap name for (read) IO\n"
+"   -i infile\n"
```
Comment (tchaikov) [inline src/tools/rados/rados.cc:192]: could you instead use a long name, like `--input-file`, for better readability if we are going to use this option in scripts. and might want to add a short description for this option?

**Comment by Songweibin** [comment]
@tchaikov  Updated, thank you.

File: src/tools/rados/rados.cc [hunk 2]
```diff
@@ -189,6 +189,8 @@ void usage(ostream& out)
 "   -s name\n"
 "   --snap name\n"
 "        select given snap name for (read) IO\n"
+"   --input-file infile\n"
+"        specify the val to use for object's object map from input file\n"
```
Comment (tchaikov) [inline src/tools/rados/rados.cc:193]:
```suggestion
"        use the content of the specified file in place of <val>"
```

File: src/tools/rados/rados.cc [hunk 3]
```diff
@@ -189,6 +189,8 @@ void usage(ostream& out)
 "   -s name\n"
 "   --snap name\n"
 "        select given snap name for (read) IO\n"
+"   --input-file infile\n"
```
Comment (tchaikov) [inline src/tools/rados/rados.cc:192]:
```suggestion
"   --input-file file\n"
```

**Comment by Songweibin** [comment]
@tchaikov  Done, thank you.

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-07-15_09:26:30-rados-wip-kefu2-testing-2020-07-15-1237-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-07-16_07:05:31-rados-wip-kefu2-testing-2020-07-16-1151-distro-basic-smithi/

**Pull request merged**